go.mod

```diff
@@ -16,6 +16,7 @@ require (
 	github.com/cespare/xxhash/v2 v2.2.0 // indirect
 	github.com/matttproud/golang_protobuf_extensions/v2 v2.0.0 // indirect
 	github.com/oschwald/maxminddb-golang v1.11.0 // indirect
+	github.com/patrickmn/go-cache v2.1.0+incompatible
```

Comment:
Import is in the wrong place..  Also the module hasn't been updated since 2017. Might be worth checking something else out.